### ISSUE TITLE
Remove test case Verifier 11 

### ISSUE THREAD

**Issue by OR13** [w3c-ccg/vc-http-api#82]
11. The Verifier's Verify Credential HTTP API MUST adhere to the proof verification format.

This is essentially a dupe fo the first test... it amounts to "does verification succeed".

**Comment by OR13** [comment]
may also be a dupe of `13. The Verifier\'s Verify Credential HTTP API MUST support the verification of, JSON-LD Proof, Ed25519Signature2018.`

**Comment by mavarley** [comment]
Agreed it may be a dupe (not sure if it is catching a subtly or not) but does removing this req. make the test suite easier to implement? Or is it just "removing noise" from the reqs?
(just wondering why it is marked 'critical').

**Comment by OR13** [comment]
@mavarley yes, changes to the test suite need comments! I marked them as critical before we had a release target.
